### PR TITLE
633 save stop ui fixes

### LINKED ISSUE #633: Save stop info in BE
BE:
https://gitlab.hsl.fi/jore/jore-map-backend/merge_requests/26

FE:
https://github.com/HSLdevcom/jore-map-ui/pull/655

Front end changes are mostly just bug fixes

### PR DESCRIPTION
**Pull request by kronken**
Closes #633 together with BE changes.

This PR fixes problem that made it possible to save stops without radius.

### CODE REVIEW

File: src/validation/models/stopValidationModel.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 const nodeValidationModel = {
```
Comment (Divinare) [inline src/validation/models/stopValidationModel.ts]: stopValidationModel

File: src/components/map/layers/mapIcons/NodeMarker.tsx [hunk 2]
```diff
@@ -197,7 +197,7 @@ class NodeMarker extends Component<INodeMarkerProps> {
                     && this.onMoveMarker('coordinates')}
                 >
                 {
-                    displayCircle
+                    displayCircle && Boolean(this.props.node.stop!.radius)
```
Comment (Divinare) [inline src/components/map/layers/mapIcons/NodeMarker.tsx]: Hmm, if node has no radius, before it displayed circle with default radius?

I would refactor a bit and say:

renderStopRadiusCircle()

```
    private renderStopRadiusCircle = () => {
        const stop = this.props.node.stop;
        if (!stop) throw new Error('Stop not found'); // Should never occur
        const radius = stop.radius ? stop.radius : DEFAULT_RADIUS;
        return (
            <Circle
                className={s.stopCircle}
                center={this.props.node.coordinates}
                radius={radius}
            />
        );
    }
```

**Comment by Divinare** [comment]
Editing of radius as " " (empty space) seems to throw error to console

Making radius empty seems to throw warning

**Comment by Divinare** [comment]
Saving stop with type "Pois käytöstä" makes the routePath completely unusable (can't fetch it anymore...)

Wasn't broken here I guess (not sure though) but fix here if its easy?

**Comment by kronken** [comment]
Fixed.

**Comment by Divinare** [comment]
Seems to work, review ok.

Added cards: #660 #661

**Pull request merged**